src/data/meetings.json

```diff
@@ -133,6 +133,16 @@
     "localShare": null,
     "remoteShare": null
   },
+  "remoteMediaWaiting": {
```

Comment:
Could you reuse one of the existing meeting objects? This looks like it can be swapped with `noMedia`

Reply:
Implemented this change.